src/main/java/com/google/sps/data/DeliverySlotManager.java

```diff
@@ -22,6 +22,7 @@
 import com.google.maps.model.LatLng;
 import com.google.maps.errors.ApiException;
 import java.io.IOException;
+import java.sql.Time;
```

Comment:
https://www.baeldung.com/java-8-date-time-intro

LocalTime and LocalDate are the new variants of time/date structures to be used in java it seems.

Using packages from `sql` definitely seems odd, since you're not dealing with sql here.

Reply:
Right, I will change startTime and endTime to LocalTime. 
Since java.time.LocalDate can't be stored in datastore, is it okay to use java.util.Date for startTime and endTime in the DeliverySlot class?